edx_proctoring/api.py

```diff
@@ -1715,6 +1715,8 @@ def _calculate_allowed_mins(exam, user_id):
             # e.g current_datetime=09:00, due_datetime=10:00 and allowed_mins=120(2hours)
             # then allowed_mins should be 60(1hour)
             allowed_time_limit_mins = int((due_datetime - current_datetime).total_seconds() / 60)
```

Comment:
nit: this could also be `max(int((due_datetime - current_datetime).total_seconds() / 60), 0)`